frontend/app/src/components/StreamlitDialog/ThemeCreatorDialog.tsx

```diff
@@ -162,10 +162,6 @@ refresh the page.`}
             value={secondaryBackgroundColor}
           />
 
-          <StyledFullRow>
-            <ThemeOption name="font" value={String(themeInput.font)} />
```

Comment:
Removed this for now since setting the old font parameter is deprecated. We might want to add it at some later stage to support the new advanced font options.